example/package.json

```diff
@@ -6,7 +6,6 @@
     "android": "npx react-native run-android",
     "ios": "npx react-native run-ios",
     "start": "react-native start",
-    "test": "jest __tests__",
```

Comment:
**suggestion:** Since it's no longer used in example app, let's remove `jest` from `devDependencies`, and all it's related dependencies and config.

Reply:
Detox still uses Jest, so we still need it.